src/pages/groupComparison/GroupComparisonUtils.ts

```diff
@@ -192,38 +194,43 @@ export function getNumSamples(
 export function getOverlapFilteredGroups(
     groups:ComparisonGroup[],
     info:{
-        overlappingSamplesSet:ListIndexedMap<boolean>,
-        overlappingPatientsSet:ListIndexedMap<boolean>
+        overlappingSamplesSet:ListIndexedSet,
+        overlappingPatientsSet:ListIndexedSet
     }
 ) {
     // filter out overlap
     const overlappingSamplesSet = info.overlappingSamplesSet;
     const overlappingPatientsSet = info.overlappingPatientsSet;
+
     return groups.map(group=>{
         let hasOverlappingSamples = false;
         let hasOverlappingPatients = false;
         const studies = [];
         for (const study of group.studies) {
             const studyId = study.id;
-            studies.push({
-                id: studyId,
-                samples: study.samples.filter(sampleId=>{
-                    if (overlappingSamplesSet.has(studyId, sampleId)) {
-                        hasOverlappingSamples = true;
-                        return false;
-                    } else {
-                        return true;
-                    }
-                }),
-                patients: study.patients.filter(patientId=>{
-                    if (overlappingPatientsSet.has(studyId, patientId)) {
-                        hasOverlappingPatients = true;
-                        return false;
-                    } else {
-                        return true;
-                    }
-                }),
+            const samples = study.samples.filter(sampleId=>{
+                if (overlappingSamplesSet.has(studyId, sampleId)) {
```

Comment:
i was looking at documenation for has and can't find anything about multiple arguments.  what does this do?

Reply:
this is a set class that I wrote, ListIndexedSet, that uses rest args as the key